### ISSUE TITLE
Issue with latest YNAB?

### ISSUE THREAD

**Issue by dannyvankooten** [toolkit-for-ynab/unofficial-desktop-for-ynab#3]
Getting the following error when trying to launch the client.

```
Uncaught Exception:
ReferenceError: createWindow is not defined
    at EventEmitter.<anonymous> (/Applications/Unofficial Desktop for YNAB.app/Contents/Resources/app.asar/main.js:53:5)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
```

**Comment by thekevinbrown** [comment]
That looks much more like the actual election method to make a window has gone missing. That's really odd. Is this something that happens every time for you, or just from time to time?

**Comment by dannyvankooten** [comment]
Thanks for the quick response @blargity. It just started happening to me yesterday but just now I downloaded a fresh copy of your release and that one worked. I then fired up (or refocused) the app instance which was still in my dock and the error occurred again.

I did a full quit of both running instances and then fired them up again - both times it worked. So this  looks to have something to do with app focus? 

If no one else is experiencing this then it's probably something on my OS and at least I know how to workaround it now, so closing this for you. Thanks for the help!
